<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addPeriodButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add period</string>
    </property>
   </widget>
   <widget class="QComboBox" name="periodTypeComboBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Usual</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Special</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Multizone</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="consumptionLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Consumption</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>461</width>
      <height>171</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <column>
     <property name="text">
      <string>Start</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Consumption</string>
     </property>
    </column>
   </widget>
   <widget class="QDateTimeEdit" name="startPeriodDateTimeEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>dd.MM.yyyy HH:mm</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="endPeriodDateTimeEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2022</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="displayFormat">
     <string>dd.MM.yyyy HH:mm</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>End</string>
    </property>
   </widget>
   <widget class="QPushButton" name="writePushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readPushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearPushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Print</string>
    </property>
   </widget>
   <widget class="QLabel" name="printLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>pushButton</zorder>
   <zorder>addPeriodButton</zorder>
   <zorder>periodTypeComboBox</zorder>
   <zorder>consumptionLineEdit</zorder>
   <zorder>tableWidget</zorder>
   <zorder>startPeriodDateTimeEdit</zorder>
   <zorder>endPeriodDateTimeEdit</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>writePushButton</zorder>
   <zorder>readPushButton</zorder>
   <zorder>clearPushButton</zorder>
   <zorder>calculateButton</zorder>
   <zorder>resultLabel</zorder>
   <zorder>printLabel</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
